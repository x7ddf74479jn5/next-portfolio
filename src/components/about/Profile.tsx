import { WrapInSection } from "src/components/layouts/Section";
import styles from "src/styles/components/about/Profile.module.scss";
import { images } from "src/utils/images";

const Profile: React.VFC = () => {
  return (
    <div className={styles.bgPrimary}>
      <WrapInSection id={"profile"}>
        <h2 className={styles.headlineAboutWhite}>Profile</h2>
        <div className={styles.gridProfile}>
          <div className={styles.left}>
            <div className={styles.mediaProfile}>
              <img src={images.pandasharkIcon.path} alt={images.pandasharkIcon.alt} />
              <p>パンダシャーク</p>
            </div>
          </div>
          <div className={styles.right}>
            <p>喫茶店の調理職×Webエンジニア。</p>
            <p>
              珈琲と美味しい料理が好き。
              <br />
              得意な料理ジャンルはイタリアンです。
            </p>
            <p>現在、プログラミング・情報技術者資格試験の勉強と転職活動を行っています。</p>
          </div>
        </div>
      </WrapInSection>
    </div>
  );
};

export default Profile;
