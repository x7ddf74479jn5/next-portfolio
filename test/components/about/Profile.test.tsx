import { render } from "@testing-library/react";
import Profile from "src/components/about/Profile";

describe("Profile", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Profile />);
    expect(asFragment()).toMatchSnapshot();
  });
});
