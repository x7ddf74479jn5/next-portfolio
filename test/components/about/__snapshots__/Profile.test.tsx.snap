// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="section bgPrimary"
    id="profile"
  >
    <div
      class="sectionWrapIn"
    >
      <h2
        class="headlineAboutWhite"
      >
        Profile
      </h2>
      <div
        class="gridProfile"
      >
        <div
          class="left"
        >
          <div
            class="mediaProfile"
          >
            <img
              alt="Pandashark's icon"
              src="/img/icons/pandashark_icon.png"
            />
            <p>
              パンダシャーク
            </p>
          </div>
        </div>
        <div
          class="right"
        >
          <p>
            喫茶店の調理職×Webエンジニア。
          </p>
          <p>
            珈琲と美味しい料理が好き。
            <br />
            得意な料理ジャンルはイタリアンです。
          </p>
          <p>
            現在、プログラミング・情報技術者資格試験の勉強と転職活動を行っています。
          </p>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
